src/modules/getReservations.js

```diff
@@ -0,0 +1,29 @@
+const postReservation = async (itemId, username, startDate, endDate) => {
+  const response = await fetch(
+    'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tN0rUHEjucNdKTNwHcwx/reservations',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        item_idetails: itemId,
+        username,
+        startDate,
+        endDate,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    },
+  );
+  const responseJSON = await response.json();
+  return responseJSON;
+};
+
+const getReservations = async (itemId) => {
+  const response = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tN0rUHEjucNdKTNwHcwx/reservations?item_id=${itemId}`,
+  );
```

Comment:
- [ ] Kindly note that after I refresh the page and click on the reservation button to see the reservation I added, there is no reservation showing so make sure that you fix this to make your project work perfectly. See the screenshot below :point_down: 
  
  ![image](https://github.com/iamsjunaid/js-capstone-2/assets/92303891/4fc3dfb7-dcaf-40bc-815e-16b3d284ee12)